src/components/Swap/SwapSettings.tsx

```diff
@@ -116,6 +118,16 @@ const SwapSettings = ({
                         }
                         props.input.onChange(currentTarget.value as string);
                       }}
+                      onBlur={() => {
+                        if (
+                          !minDexFee ||
```

Comment:
Is it really necessary

Reply:
yes. it will give us default value if input has errors.